### DISCUSSION TITLE
[FEATURE REQUEST] "Forgot Password" functionality

### DISCUSSION THREAD

**Discussion by jeremymeyers** [jonaswinkler/paperless-ng#941] [Feature Requests]
**Describe the bug**
There is currently no way to reset a forgotten or borked password short of going into the database (Which is not documented)

**To Reproduce**
1. Go to login page
2. Enter incorrect password
3. See error with no alternative

**Expected behavior**
A "forgot password?" form which will prompt for email address associated with the account and send a reset link or a new password when submitted.

**Comment by jonaswinkler** [comment]
Django has built-in support for that, so I imagine it's not too difficult to get this implemented. The hardest part would be smtp setup, I guess.

I'm open for PRs on this one.

Also, there's always the `createsuperuser` management command, and you can use that to create a new user from the command line.
